b-locker-web/src/components/organisation/topbar/orgTopbar.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './orgTopbar.scss';
+
+const topbar: React.FC = () => {
+
+    return (
+        <section className="topnav">
+            <div className="container">
+                <a className="active" href="#home">Home</a>
+                <a href="#news">News</a>
+                <a href="#contact">Contact</a>
+                <a href="#about">About</a>
```

Comment:
Please translate these using the i18n implementation. Take a gander at TestPage to see this being used. You also need to add the translations to the en.json & nl.json files. name them something like "topnav.home.label" etc.